cmd/mdatagen/internal/templates/telemetry.go.tmpl

```diff
@@ -61,6 +63,12 @@ func (tbof telemetryBuilderOptionFunc) apply(mb *TelemetryBuilder) {
   tbof(mb)
 }
 
+func WithAttributeSet(s attribute.Set) telemetryBuilderOptionFunc {
```

Comment:
Why don't pass all the attributes with the generate `Record` functions same as we do for regular attributes?